cmd/wcloud/cli.go

```diff
@@ -29,10 +29,10 @@ var (
 
 func usage() {
 	fmt.Println(`Usage:
-	deploy <image>:<version>   Deploy image to your configured env
-	list                       List recent deployments
-	config (<filename>)        Get (or set) the configured env
-	logs <deploy>              Show lots for the given deployment`)
+	deploy (-m) <image>:<version>   Deploy image to your configured env
```

Comment:
-m probably needs a value?
isn't the commit message set in config? Or does this override it for a once-off?